taxonomies/food/ingredients.txt

```diff
@@ -17252,16 +17267,22 @@ en: refined rapeseed oil
 hr: rafinirano ulje repice
 lt: rafinuotas rapsų aliejus
 pl: rafinowany olej rzepakowy, olej rzepakowy rafinowany
+ecobalyse:en: rapeseed-oil
```

Comment:
Same here, ecobalyse:en: rapeseed-oil should only be put in the "en: rapeseed oil" entry, and it will apply to all children